content/security/docs/compliance.md

```diff
@@ -50,6 +50,11 @@ Policy can be thought of as a set of rules for governing behaviors, i.e. behavio
 
 [Kyverno](https://kyverno.io/) is a policy engine designed for Kubernetes. With Kyverno, policies are managed as Kubernetes resources and no new language is required to write policies. This allows using familiar tools such as kubectl, git, and kustomize to manage policies. Kyverno policies can validate, mutate, and generate Kubernetes resources plus ensure OCI image supply chain security. The [Kyverno CLI](https://kyverno.io/docs/kyverno-cli/) can be used to test policies and validate resources as part of a CI/CD pipeline. All the Kyverno community policies can be found on the [Kyverno website](https://kyverno.io/policies/), and for examples using the Kyverno CLI to write tests in pipelines, see the [policies repository](https://github.com/kyverno/policies).
 
++ [Datree](https://www.datree.io/) is a native Kubernetes solution for centralized policy management. When K8s resources are pushed into a cluster, Datree checks their configurations against pre-defined policies. As soon as misconfigurations are detected, Datree displays a detailed output of their location and impact, along with instructions on how to fix them and based on its settings, Datree can both audit and prevent resources that contains violations from being applied.
```

Comment:
1) it looks like datree is a feature of kyverno because of the bullet at the beginning.
2) I would add relevant links (e.g. "built-in policy for EKS", "EKS best practices", etc.)